### PR TITLE
[DOC] Update Pushing spans HTTP doc

### LINKED ISSUE #1695: [Docs] Review and update the Pushing spans with HTTP documentation
Review the [Pushing spans with HTTP](https://grafana.com/docs/tempo/next/api_docs/pushing-spans-with-http/) documentation in the API reference area. 

Tasks: 
- [X] Verify that the instructions and examples still work
- [X] Provide additional examples - Integrate with Intro to MLT, ~~how to use with Cloud Traces~~, how to use with OTLP receiver
- [X] Walk through the examples and verify that it works

This site generates tracing data. You can use it to point Tempo to gather traces instead of having to set up a data source (uses Telemetry gen): https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/cmd/telemetrygen

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:

Updates the Pushing spans with HTTP example doc to include more examples, use a Docker example, and remove Zipkin. 

**Which issue(s) this PR fixes**:
Fixes #1695 #2522 

**Checklist**
- [ ] Tests updated
- [X] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/tempo/api_docs/pushing-spans-with-http.md [hunk 1]
```diff
@@ -9,101 +9,126 @@ aliases:
...
-storage:
-  trace:
-    backend: local
-    local:
-      path: /tmp/tempo/blocks
-```
+## Start Tempo using the quick start
 
-and run Tempo using it:
+Use the instructions in the [Quick start for Tempo documentation]({{< relref "../getting-started/docker-example/" >}}) to start a local instance of Tempo and Grafana.
```
Comment (knylander-grafana) [inline docs/sources/tempo/api_docs/pushing-spans-with-http.md:23]:
```suggestion
Use the instructions in the [Quick start for Tempo documentation]({{< relref "../getting-started/docker-example" >}}) to start a local instance of Tempo and Grafana.
```

File: docs/sources/tempo/api_docs/pushing-spans-with-http.md [hunk 2]
```diff
@@ -9,101 +9,126 @@ aliases:
...
+{"batches":[{"resource":{"attributes":[{"key":"service.name","value":{"stringValue":"my.service"}}]},"scopeSpans":[{"scope":{"name":"my.library","version":"1.0.0"},"spans":[{"traceId":"W47/95gDgQPSabYzgT/HAA==","spanId":"7uGbfsPBsQA=","name":"I am a span!","kind":"SPAN_KIND_SERVER","startTimeUnixNano":"1689969302000000000","endTimeUnixNano":"1689970000000000000","attributes":[{"key":"my.span.attr","value":{"stringValue":"some value"}}],"status":{}}]}]}]}
 ```
 
-However, staring at a json blob in bash is not very fun.  Let's start up Tempo query so we can visualize our trace.  Tempo query is [Jaeger Query](https://hub.docker.com/r/jaegertracing/jaeger-query/) with a [GRPC Plugin](https://github.com/jaegertracing/jaeger/tree/master/plugin/storage/grpc) that allows it to query Tempo.
+1. Copy and paste the updated `curl` command into a terminal window.
 
-```bash
-docker run --env BACKEND=localhost:3200 --net host grafana/tempo-query:latest
-```
+### Use TracedQL to search for a trace
```
Comment (knylander-grafana) [inline docs/sources/tempo/api_docs/pushing-spans-with-http.md:111]:
```suggestion
### Use TraceQL to search for a trace
```

File: docs/sources/tempo/api_docs/pushing-spans-with-http.md [hunk 3]
```diff
@@ -9,101 +9,126 @@ aliases:
...
 
-And open `http://localhost:16686/trace/0123456789abcdef` in the browser of your choice to see:
+Alternatively, you can also use [TraceQL]({{< relref "../traceql" >}}) to also search for the trace that was pushed.
```
Comment (knylander-grafana) [inline docs/sources/tempo/api_docs/pushing-spans-with-http.md:113]:
```suggestion
Alternatively, you can also use [TraceQL]({{< relref "../traceql" >}}) to search for the trace that was pushed.
```

File: docs/sources/tempo/api_docs/pushing-spans-with-http.md [hunk 4]
```diff
@@ -9,101 +9,126 @@ aliases:
...
+```bash
+curl -G -s http://localhost:3200/api/search --data-urlencode 'q={ .service.name = "my.service" }'
 
-## More spans
+{"traces":[{"traceID":"5b8efff798038103d269b633813fc700","rootServiceName":"my.service","rootTraceName":"I am a span!","startTimeUnixNano":"1694718625557000000","durationMs":10000,"spanSet":{"spans":[{"spanID":"eee19b7ec3c1b100","startTimeUnixNano":"1694718625557000000","durationNanos":"10000000000","attributes":[{"key":"service.name","value":{"stringValue":"my.service"}}]}],"matched":1},"spanSets":[{"spans":[{"spanID":"eee19b7ec3c1b100","startTimeUnixNano":"1694718625557000000","durationNanos":"10000000000","attributes":[{"key":"service.name","value":{"stringValue":"my.service"}}]}],"matched":1}]}],"metrics":{"inspectedBytes":"292781","completedJobs":1,"totalJobs":1}}
+```
 
-Now that we have the basics down, it's easy to continue building our trace.  By specifying the same trace ID and a parent span ID, we can start building a trace.
+To format this in a more human-readable output, consider using a [tool such as `jq`](https://jqlang.github.io/jq/), which  lets you to run the same `curl` command and pipe it to `jq` to format the block, for example:
```
Comment (knylander-grafana) [inline docs/sources/tempo/api_docs/pushing-spans-with-http.md:122]:
```suggestion
To format this in a more human-readable output, consider using a [tool such as `jq`](https://jqlang.github.io/jq/), which lets you to run the same `curl` command and pipe it to `jq` to format the block. For example:
```

File: docs/sources/tempo/api_docs/pushing-spans-with-http.md [hunk 5]
```diff
@@ -9,101 +9,126 @@ aliases:
...
+	}]
+}'
 ```
 
-Note that the `timestamp` field is in microseconds and was obtained by running `date +%s%6N`.  The `duration` field is also in microseconds and so 100000 is 100 milliseconds.
+Note that the `startTimeUnixNano` field is in nanoseconds and can be obtained by any tool that provides the epoch date in nanoseconds (for example, under Linux, `date +%s%8N`). The `endTimeUnixNano` field is also in nanoseconds, where 100000000 nanoseconds is 100 milliseconds.
+
+1. Copy and paste the curl command into a text editor.
+
+1. Replace `startTimeUnixNano` and endTimeUnixNano with current values for the last 24 hours to allow you to search for them using a 24 hour relative time range. You can get this in seconds and milliseconds from the following link.
```
Comment (mapno) [inline docs/sources/tempo/api_docs/pushing-spans-with-http.md:79]:
```suggestion
1. Replace `startTimeUnixNano` and `endTimeUnixNano` with current values for the last 24 hours to allow you to search for them using a 24 hour relative time range. You can get this in seconds and milliseconds from the following link.
```

File: docs/sources/tempo/api_docs/pushing-spans-with-http.md [hunk 6]
```diff
@@ -9,101 +9,126 @@ aliases:
...
+   1. Select Run query to list available traces.
+   1. Select the trace ID (yellow box) to view details about the trace and its spans.
+
+![Using the TraceQL query builder on Explore to view pushed trace in Grafana.](/static/img/docs/tempo/push-spans-search-span-grafana.png "View the span in Grafana")
 
 ## Retrieve traces
 
 The easiest way to get the trace is to execute a simple curl command to Tempo.  The returned format is [OTLP](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/trace/v1/trace.proto).
 
+1. Replace the trace ID in the curl command with the trace ID that was generated from the push (you could use Grafana’s Explorer page to find this, as shown in the previous section).
```
Comment (mapno) [inline docs/sources/tempo/api_docs/pushing-spans-with-http.md:101]: It's also in the data that's sent with `curl`.

**Review by mapno — APPROVED** [review]
Nice!

File: docs/sources/tempo/api_docs/pushing-spans-with-http.md (hunk 6 above)
Comment (knylander-grafana) [inline docs/sources/tempo/api_docs/pushing-spans-with-http.md:101]:
```suggestion
1. Replace the trace ID in the `curl` command with the trace ID that was generated from the push (you could use Grafana’s Explorer page to find this, as shown in the previous section).
```

File: docs/sources/tempo/getting-started/docker-example.md [hunk 7]
```diff
@@ -15,18 +15,20 @@ The Tempo examples running with Docker using docker-compose all include a versio
...
 To learn more, read the [local storage example README](https://github.com/grafana/tempo/blob/main/example/docker-compose/local).
 
-## Alternative: Complete MLT example
+## Alternative: Complete MLTP example
 
-If you would like to use a demo with Tempo, Mimir, Loki and Grafana, then try the [Introduction to Metrics, Logs and Traces in Grafana](https://github.com/grafana/intro-to-mlt).
-Intro-to-mlt provides a self-contained environment for learning about Mimir, Loki, Tempo, and Grafana.
-It includes detailed explanations of each component and annotated configurations for a single-instance deployment.
-Data from intro-to-mlt can also be pushed to Grafana Cloud.
+If you would like to use a demo with Tempo, Mimir, Loki, and Grafana, then try the [Introduction to Metrics, Logs, Traces and Profiling in Grafana](https://github.com/grafana/intro-to-mlt).
```
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/docker-example.md:20]:
```suggestion
If you would like to use a demo with Tempo, Mimir, Loki, and Grafana, then try the [Introduction to Metrics, Logs, Traces, and Profiling in Grafana](https://github.com/grafana/intro-to-mlt).
```

File: docs/sources/tempo/api_docs/pushing-spans-with-http.md [hunk 8]
```diff
@@ -9,101 +9,126 @@ aliases:
...
+   ```
+
+1. Copy the updated curl command to a terminal window and run it.
+
+1. View the trace in Grafana:
+   1. Open a browser window to http://localhost:3000.
+   1. Open the Explorer page and select the Tempo data source.
+   1. Select the Search query type.
+   1. Select Run query to list available traces.
+   1. Select the trace ID (yellow box) to view details about the trace and its spans.
```
Comment (knylander-grafana) [inline docs/sources/tempo/api_docs/pushing-spans-with-http.md:93]:
```suggestion
1. View the trace in Grafana:
   1. Open a browser window to http://localhost:3000.
   1. Open the **Explorer** page and select the Tempo data source.
   1. Select the **Search** query type.
   1. Select **Run query** to list available traces.
   1. Select the trace ID (yellow box) to view details about the trace and its spans.
```

File: docs/sources/tempo/api_docs/pushing-spans-with-http.md (hunk 6 above)
Comment (knylander-grafana) [reply docs/sources/tempo/api_docs/pushing-spans-with-http.md:101]: I've added this to text. Thank you.

**Comment by knylander-grafana** [comment]
Thank you, @mapno

**Pull request merged**